test/integration/authz_test.go

```diff
@@ -47,5 +47,5 @@ func TestValidAuthzExpires(t *testing.T) {
 	// The authz should have the expected expiry date
 	expectedExpires := time.Now().AddDate(0, 0, validAuthorizationLifetime).Round(time.Minute)
 	actualExpires := authzOb.Expires.Round(time.Minute)
-	test.AssertEquals(t, expectedExpires.Equal(actualExpires), true)
+	test.AssertEquals(t, expectedExpires, actualExpires)
```

Comment:
Using `==` for checking time equality is a pitfall, as it compares not just the time but the monotonic clock and location as well, much safer to use `Equal`.